src/reducers/account/index.js

```diff
@@ -146,6 +148,51 @@ const account = handleActions({
     })
 }, initialState)
 
+const ledger = handleActions({
```

Comment:
should this go into a separate reducer module? Doesn't seem to be related to single account.

Reply:
sure